### PR TITLE
Moving timezone migration from SamEcoreUserManagement

### PR DESCRIPTION
**Pull request by tchevily**
Moving migration from sam-ecore-user-management to sam-ecore

[https://github.com/CanalTP/NMM/pull/263](https://github.com/CanalTP/NMM/pull/263)

### CODE REVIEW

**Comment by tchevily** [comment]
depends on
[https://github.com/CanalTP/SamEcoreUserManagerBundle/pull/13](https://github.com/CanalTP/SamEcoreUserManagerBundle/pull/13)

File: Migrations/pdo_pgsql/Version010.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    /**
+     * Adding timezone column into user table
+     *
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $this->addSql("ALTER TABLE public.t_user_usr ADD COLUMN usr_timezone varchar(255) DEFAULT 'Europe/Paris'");
```
Comment (nberard) [inline Migrations/pdo_pgsql/Version010.php]: we need to hydrate each existing user with this timezone or the code breaks (it supposes there is always a valid timezone in this column)

Comment (tchevily) [inline Migrations/pdo_pgsql/Version010.php]: usr_timezone is a new column, so there would be no issue with this addition.
The problem could occure if you already have this column into t_user_usr table.

Comment (nberard) [inline Migrations/pdo_pgsql/Version010.php]: i added the NOT NULL constraint to force default value on existing rows and i updated the fixtures management not to create user without timezone

**Pull request merged**